src/components/common/Layout/components/Content.tsx

```diff
@@ -0,0 +1,10 @@
+import { Box, BoxProps } from "@chakra-ui/react"
+import { PropsWithChildren } from "react"
+
+type Props = PropsWithChildren<Omit<BoxProps, "title">>
+
+const Content = ({ children, ...wrapperProps }: Props) => (
```

Comment:
I think we should just abstract it into MainContent, if we need it at all, but I'm not sure about that as that container is just a simple div too. I'd remove for now and add back to MainContent later when applying the new layout for other pages too, if we happen to need it